docs/source/conf.py

```diff
@@ -117,7 +117,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+# html_static_path = ['_static']
```

Comment:
Is this just cleanup, or is this change necessary for the code to work?

Reply:
``sphinx`` just trows a warning that the folder ``_static`` does't exist if ``html_static_path`` is set.

Reply:
So clean-up then. Thanks!